server/routes/visitMigration/startVisitsMigrationValidator.test.ts

```diff
@@ -1,30 +1,13 @@
-import { Request } from 'express'
 import type { StartVisitsMigrationForm } from 'forms'
 import validate from './startVisitsMigrationValidator'
 
 describe('startVisitsMigrationValidator', () => {
-  const req = {
```

Comment:
validator changed so it just returns a list of errors rather than doing `req.flash` and returning next url. it was just too clunky